UnitsNet/Scripts/UnitDefinitions/Pressure.json

```diff
@@ -8,7 +8,7 @@
             "PluralName": "Pascals",
             "FromUnitToBaseFunc": "x",
             "FromBaseToUnitFunc": "x",
-            "Prefixes": ["Kilo","Mega"],
+            "Prefixes": ["Micro","Deca","Hecto","Kilo","Mega","Giga"],
```

Comment:
Any particular reason why Milli and Centi is left out? I don't know what prefixes are commonly used with Pascal.